hack/verify-symbols.sh

```diff
@@ -30,14 +30,15 @@ BADSYMBOLS=(
   "httptest"
   "testify"
   "testing[.]"
+  "TrackStorageCleanup"
 )
 
 # b/c hyperkube binds everything simply check that for bad symbols
-SYMBOLS="$(nm "${KUBE_OUTPUT_HOSTBIN}/hyperkube")"
```

Comment:
```sh
GOLDFLAGS="-w" make -C "${KUBE_ROOT}" WHAT=cmd/hyperkube
```

Might do the trick. Looks like `GOLDFLAGS` defaults to `-s -w` in `kubernetes/hack/lib/golang.sh` where the `-s` strips out symbols.

The only problem with this is that we leave a `_output/local/bin/linux/amd64/hyperkube` binary in the project that doesn't have symbols stripped. If we can build a custom output directory that might be safer..

Reply:
fixed